markdownPreview/index.ts

```diff
@@ -1,10 +1,18 @@
 import mermaid, { MermaidConfig } from 'mermaid';
 import { renderMermaidBlocksInElement } from './mermaid';
+import * as logos from '@iconify-json/logos';
 
-function init() {
+function init() { 
     const configSpan = document.getElementById('markdown-mermaid');
     const darkModeTheme = configSpan?.dataset.darkModeTheme;
     const lightModeTheme = configSpan?.dataset.lightModeTheme;
+  
+    mermaid.registerIconPacks([
+      {
+        name: 'logos',
+        loader: () => Promise.resolve(logos.icons),
```

Comment:
Can this be a lazy import so it's unload loaded if needed?

Reply:
Yes! 
I have changed to lazy import !